### ISSUE TITLE
we need the a variable to replace 50 in the epochs in matchdg

### ISSUE THREAD

**Issue by smilesun** [marrlab/DomainLab#187]
https://github.com/marrlab/DomainLab/blob/214f667fc005ea3a4431b194b9618fecbb9952f3/examples/yaml/benchmark_mnist.yaml#L126

    # this shall be set to be epos - epochs_ctr
    # as epos cannot be referenced here one needs to hardcode this right number of epos here
    epochs_erm:
      reference: max(1, 50 - epochs_ctr)

Note we have
```
domainlab_args:
  # Domainlab arguments passed to each task.
  # task specific arguments take precedence.
  ## dataset
  task: mnistcolor10
  tr_d: [1, 2, 4]
  dmem: False
  lr: 0.001
  epos: 50
  es: 5
  bs: 64
  nname: conv_bn_pool_2
  san_check: True
```

where epos are defined

**Comment by smilesun** [comment]
In https://github.com/marrlab/DomainLab/pull/188

**Comment by smilesun** [comment]
fixed c119559f73547a4463a07d00c9b116699f8e1a1c
